tests/ui/openid_connect_mock.rs

```diff
@@ -106,14 +161,34 @@ type KnownUsers = HashMap<KnownUserId, KnownUser>;
 const DEFAULT_TOKEN_DURATION_SECS: u32 = 3600;
 static MOCK_OPENID_CONNECT_SERVER_RUNNING_FLAG: AtomicBool = AtomicBool::new(false);
 
-pub async fn start() -> Option<task::JoinHandle<()>> {
+#[tokio::main]
+pub async fn main() {
+    // Log to stdout.
+    #[cfg(feature = "multi-user")]
+    let _logger = fern::Dispatch::new().format(move |out, message, record| {
+            out.finish(format_args!(
+                "{} [{}] {}",
+                chrono::Local::now().format("%Y-%m-%d %H:%M:%S"),
+                record.level(),
+                message
+            ))
+    })
+    .level(log::LevelFilter::Trace)
+    .chain(std::io::stdout())
+    .apply()
+    .map_err(|e| format!("Failed to init stderr logging: {}", e));
```

Comment:
This would only when run as a stand alone mock server with `--example` and only when logging to stdout, so you would see this in the console where you started that. All other uses of the mock server would not go through `main`

I included this mainly to silence clippy, btw.